src/controller/highlightController.js

```diff
@@ -0,0 +1,141 @@
+import models from '../models';
+import Error from '../utilities/Error';
+
+const { sequelize, Highlights, HighlightComments } = models;
+
+/**
+ * Class representing the Highlight controller
+ * @class HighlightController
+ * @description Highlight controller
+ */
+class HighlightController {
+  /**
+   * Create Highlight
+   * Route: POST: /highlights/:articleId
+   * @param {object} req - Request object
+   * @param {object} res - Response object
+   * @returns {res} res - Response object
+   * @memberof HighlightController
+   */
+  static createHighlight(req, res) {
+    const { articleId } = req.params;
+    const { commentText, comment } = req.body;
+    const userId = req.decoded.id;
+    return sequelize.transaction(async (t) => {
+      try {
```

Comment:
Use a more descriptive variable name. see line 24.

Reply:
it's fixed, thanks.